### PR TITLE
Full dokku deployment example

### PR DESCRIPTION
**Pull request by iloveitaly**

### CODE REVIEW

**Comment by josegonzalez** [comment]
Some minor lint issues :)

**Comment by iloveitaly** [comment]
@josegonzalez fixed!

File: README.md [hunk 1]
```diff
@@ -697,6 +697,52 @@ Manage storage for dokku applications
...
+  tasks:
+    - name: create app
+      dokku_app:
+        # change this name in your template!
+        app: &appname appname
+    - name: environment configuration
+      dokku_config:
+        app: *appname
+        config:
+          DOKKU_LETSENCRYPT_EMAIL: email@domain.com
```
Comment (josegonzalez) [inline README.md]: Can we change `domain.com` to `example.com`?

Comment (iloveitaly) [reply README.md]: @josegonzalez done!

File: example.yml [hunk 2]
```diff
@@ -35,3 +35,46 @@ examples:
...
+              app: *appname
+              config:
+                DOKKU_LETSENCRYPT_EMAIL: email@example.com
+                # specify port so `domains` can setup the port mapping properly
+                PORT: "5000"
+          - name: git clone
+            # note you'll need to add a deployment key to the GH repo if it's private!
+            dokku_clone:
+              app: *appname
+              repository: git@github.com:githubname/app.git
```
[the remark this answers is not in the document]
Comment (iloveitaly) [reply example.yml]: Done!

**Pull request merged**